R/get_outcomes.R

```diff
@@ -0,0 +1,126 @@
+#' Get scenario outcomes
+#'
+#' @param x A `scenario` or `comparison` object with the scenario outcome data
+#' prepared. Check for whether data has been prepared using [sce_has_data()].
+#'
+#' @return A single `data.table` holding the output of all replicates of the
+#' scenario. If `x` is a `comparison` object, the `data.table` holds the
+#' outcomes of each replicate of each scenario specified therein. Scenarios in
+#' this case are identified either by their names if any, or by a simple
+#' synthetic identifier such as 'scenario_01'.
+#' @export
+#'
+#' @examples
+#' # For the 'scenario' class
+#' # create a scenario specification
+#' scenario_pandemic_flu <- scenario(
+#'   model_function = "finalsize::final_size",
+#'   parameters = make_parameters_finalsize_UK(),
+#'   replicates = 3 # note extra replicates
+#' )
+#'
+#' # run scenario
+#' scenario_pandemic_flu <- run_scenario(scenario_pandemic_flu)
+#'
+#' # get outcomes
+#' sce_get_outcomes(scenario_pandemic_flu)
+#'
+#' # For the 'comparison' class
+#' # prepare two scenarios of the final size of an epidemic
+#' pandemic_flu <- scenario(
+#'   model_function = "finalsize::final_size",
+#'   parameters = make_parameters_finalsize_UK(r0 = 1.5),
+#'   replicates = 1L
+#' )
+#'
+#' covid19 <- scenario(
+#'   model_function = "finalsize::final_size",
+#'   parameters = make_parameters_finalsize_UK(r0 = 5.0),
+#'   replicates = 1L
+#' )
+#'
+#' # create a comparison object
+#' a <- comparison(
```

Comment:
A more informative name for `a` would be better.

Reply:
Fixed in https://github.com/epiverse-trace/scenarios/pull/12/commits/c18de0dfcc3514a9db5fb534f22361921240a2fd